src/location.rs

```diff
@@ -96,4 +104,20 @@ impl DstLocation {
             Self::Node(_) | Self::Section(_) | Self::Direct => Err(RoutingError::BadLocation),
         }
     }
+
+    /// Returns whether the given name of the given prefix is part of this location.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `prefix` does not match `name`.
+    pub fn contains(&self, name: &XorName, prefix: &Prefix<XorName>) -> bool {
```

Comment:
function name better to be is_dst ?

Reply:
Same as https://github.com/maidsafe/routing/pull/2084#discussion_r417312930